rabbitmq.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
all this should go in the defaults folder if any of it needs to be updated. This may be useful inside a readme.md file though so peopple can see how it works.